internal/endtoend/testdata/wasm_plugin_sqlc_gen_greeter/sqlc.json

```diff
@@ -17,7 +17,7 @@
     {
       "name": "greeter",
       "wasm": {
-        "url": "https://github.com/kyleconroy/sqlc-gen-greeter/releases/download/v0.1.0/sqlc-gen-greeter.wasm",
+        "url": "https://github.com/sqlc-dev/sqlc-gen-greeter/releases/download/v0.1.0/sqlc-gen-greeter.wasm",
```

Comment:
Verified this works: https://github.com/sqlc-dev/sqlc-gen-greeter/releases/download/v0.1.0/sqlc-gen-greeter.wasm